README.md

```diff
@@ -28,28 +28,26 @@ mllam-exps
 │   ├── datastore.zarr  # output from the prepare dataset stage (not there initially)
 │   ├── graph  # output from the create graph stage (not there initially)
 │   ├── config.yaml  # configuration file for the Neural-LAM project
-│   ├── datastore.yaml  # configuration file for the data preparation stage via mllam-data-prep
-│   ├── training_params.yaml  # configuration file for the training stage
-│   └── evaluate_params.yaml  # configuration file for the evaluation stage
+│   └── datastore.yaml  # configuration file for the data preparation stage via mllam-data-prep
 ├── logs  # log files from SLURM jobs
 ├── machines  # machine configurations for the Neural-LAM project
 │   ├── environment.sh  # source environment incl. python modules
-│   ├── secrets.sh  # file containing some user specific secrets (DO NOT COMMIT!)
-│   └── slurm.evaluate.sh  # SLURM wrapper script for evaluation
+│   ├── check_for_venv_path.sh  # script to check if path to virtual python environment is set 
+│   ├── slurm.neural-lam.sh  # SLURM wrapper script for training and evaluation
+│   └── slurm.mllam-data-prep.sh  # SLURM wrapper script for mllam-data-prep
 ├── dvc.lock  # lock file for DVC containing checksums for the latest stages
 ├── dvc.yaml  # DVC pipeline definition
-├── train_wrapper.py  # wrapper script to allow neural-lam parameters be passed via DVC/params.yaml
+├── params.yaml  # Parameters to train and evaluate model, which are converted to command line arguments to neural_lam
 ├── version.mllam.txt  # Lock file for neural-lam version (external dependencies are not well supported by DVC)
 ```
 
 ## Setup
 To setup DVC, DVC needs to be installed:
 ```bash
-pip install dvc --target packages_dvc
-export PYTHONPATH=packages_dvc:$PYTHONPATH
+pip install dvc
 ```
 
-To be able to track experiments with MLflow, add your username and password to the file machines/secrets.sh:
+To be able to track experiments with MLflow, add your username and password to a `~/.bashrc`:
```

Comment:
Why not inside `secrets.sh` anymore?

Reply:
because these can't be used in queue jobs. The reason being that `secrets.sh` isn't tracked in the git repo and only tracked files are copied across to the queued runs. So we had to switch to requiring the user to set these venv vars ahead of time (rather than sourcing `secrets.sh` in the slurm job submission scripts)